tests/RulesetTests.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
I get it's verbose to have too many lines (and empty lines) boilerplate at the start of every file. It would also be unexpected to find anything after `<?php` however